src/shared/lib/pathwayMapper/PathwayMapperTable.tsx

```diff
@@ -206,6 +206,11 @@ export default class PathwayMapperTable extends React.Component<
 
     render() {
         const columns = _.sortBy(this._columns);
+        this.dataStore.page = findPageOfSelectedPathway(
+            this.props.data,
+            this.dataStore,
+            this.props.selectedPathway
+        );
```

Comment:
Ideally render method shouldn't update any store variable. This may cause side effects.